docs/make.jl

```diff
@@ -1,23 +1,19 @@
 using Neuroblox
 using Documenter
 
-cp("./docs/Manifest.toml", "./docs/src/assets/Manifest.toml", force = true)
-cp("./docs/Project.toml", "./docs/src/assets/Project.toml", force = true)
-
 DocMeta.setdocmeta!(Neuroblox, :DocTestSetup, :(using Neuroblox); recursive = true)
 
 include("pages.jl")
 
 makedocs(sitename = "Neuroblox",
     authors = "Neuroblox Inc.",
     modules = [Neuroblox],
-    clean = true, doctest = false, linkcheck = true,
+    clean = true, doctest = false, linkcheck = false,
     warnonly = [:docs_block, :missing_docs],
     format = Documenter.HTML(assets = ["assets/favicon.ico"]),
-        #canonical = "https://docs.sciml.ai/LinearSolve/stable/"),
     pages = pages)
```

Comment:
Why was this turned off?

Reply:
It was giving an error on a couple of paper links, I only changed it to test for now, eventually it should be switched back.

Reply:
Can you just white list the bad links?

Reply:
Yep, will check them again when we sort out the workflow and if need be will add them to `linkcheck_ignore`.